ere-guests/stateless-validator/reth/guest/src/guest.rs

```diff
@@ -31,6 +31,9 @@ pub fn ethereum_guest<S: SDK>() {
     S::cycle_scope(ScopeMarker::Start, "public_inputs_preparation");
     let header = input.block.header().clone();
     let parent_hash = input.block.parent_hash;
+    let versioned_hashes_hash = calculate_versioned_hashes_hash(&input.chain_config, &input.block);
```

Comment:
This is the generic Reth guest program used for all zkVMs. Should feel quite natural what we're doing here.